src/Devices/Base/ConfigurationHandler.cs

```diff
@@ -20,11 +21,17 @@ public static string getConfig(string section, string key)
                 .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true);
 
             IConfigurationRoot configuration = builder.Build();
-            // configurationSection.Key => FilePath
-            // configurationSection.Value => C:\\temp\\logs\\output.txt
+
             IConfigurationSection configurationSection = configuration.GetSection(section).GetSection(key);
             return configurationSection.Value;
 
         }
+
+        public static string GetDeviceRuntimeSettings(string configurationSetting)
+        {
+            // TODO: put some error handling around the file access 
```

Comment:
Please, remove the TODO, fix it, or create a work item for fixing it & link it here.

Reply:
Added error handling and validation of device config file